shell/common/api/electron_api_asar.cc

```diff
@@ -63,7 +63,7 @@ class Archive : public gin::Wrappable<Archive> {
         case asar::HashAlgorithm::SHA256:
           integrity.Set("algorithm", "SHA256");
           break;
-        default:
+        case asar::HashAlgorithm::NONE:
           NOTREACHED();
```

Comment:
as before, should we `CHECK` or otherwise fail here, or is a `DCHECK` sufficient?